src/components/ChatEntry.js

```diff
@@ -1,22 +1,38 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+//spread 
+//TimeStamp has time prop
+const ChatEntry = ({ updateChat, ...message}) => {
+
+  const onLikeClick = () => {
+    updateChat({ ...message, liked: !message.liked });
```

Comment:
Nice use of your passed-in callback to allow parent-owned logic to run when the like button is clicked.